src/ArraySetopsMsg.chpl

```diff
@@ -168,6 +170,7 @@ module ArraySetopsMsg
              st.addEntry(vname, new shared SymEntry(aV));
 
              var s = try! "created " + st.attrib(vname);
```

Comment:
i see you removed the `try!` in other throwing `proc`s show we remove this one also?

Reply:
Good catch, removed these also